docker-compose.yml

```diff
@@ -98,3 +100,18 @@ services:
     volumes:
       - "./testing/docker/package-registry/config.yml:/package-registry/config.yml"
       - "./build/integrations/apm:/packages/local/apm"
+
+  metricbeat:
+    image: docker.elastic.co/beats/metricbeat:8.2.0-43df679f-SNAPSHOT
```

Comment:
will the version for this also be automatically updated when the other container versions in this file are updated?

Reply:
I looked at the update script and I believe it will be as well, yes.

Reply:
just tested this locally and it seemed to update all the service images 🙂